tests/wetlab2variations_execution_nxf_gs_input.wfex.stage

```diff
@@ -0,0 +1,45 @@
+trs_endpoint: https://workflowhub.eu/ga4gh/trs/v2/tools/
+workflow_id: 106
+version: 3
+workflow_config:
+  secure: false
+# All the inputs must be URLs or CURIEs from identifiers.org
+params:
+  general:
+    rawreads:
+      c-l-a-s-s: File
+      url:
+        - gs://gcp-public-data--broad-references/arrays/Broad_GWAS_supplemental_15061359_A1/Broad_GWAS_supplemental_15061359_A1.1.3.extended.csv
```

Comment:
Is this path the right one? Comparing it to the corresponding S3 links definition, they do not seem the same files:

https://github.com/inab/WfExS-backend/blob/02fadee6d919d30433de3a739b8f0bd8f0490d0e/tests/wetlab2variations_execution_nxf_s3_input.wfex.stage#L12

Reply:
@Acivico I take a look at your changes applied in the 752c4e2e5979034f202bacc30d7bd26c61045073 comment and it seems there are not applied well. Can you update again the file as @jmfernandez indicates in this comment, but for gs?

> Have a look at the comments I have left in the stage file. Mainly related to the gs links not pointing to the right files, by comparing the names of the pointed resources at the S3 stage file:
> 
> https://github.com/inab/WfExS-backend/blob/02fadee6d919d30433de3a739b8f0bd8f0490d0e/tests/wetlab2variations_execution_nxf_s3_input.wfex.stage#L12-L13